parliamentarian/policy.py

```diff
@@ -3,21 +3,23 @@
 import os
 import pkgutil
 import sys
-import jsoncfg
 from glob import glob
 from pathlib import Path
 
+import jsoncfg
+from jsoncfg.config_classes import ConfigJSONArray, ConfigJSONObject
+
 from . import expand_action
 from .finding import Finding
 from .misc import make_list
 from .statement import Statement
 
 
 class Policy:
-    _findings = []
-    policy_json = None
+    _findings: list[Finding] = []
```

Comment:
im not sure these class variables serve any purpose... we can probably remove them and move the typing to the __init__'s instance variables

Reply:
I don't know that they do either, but I'm inclined to leave that sort of change to a separate chunk of work.  Do we want to track that in the repository?